etc/osm-smartrak/TODO.txt

```diff
@@ -0,0 +1 @@
+Proper attribution / license
```

Comment:
comment(non-blocking): do we need to be concerned about this?

Reply:
Oh, we do but it was covered with the combination of copying the license over and citing it in the README. I'll remove this file.